src/components/Modals/CreateAssetPackModal/CreateAssetPackModal.tsx

```diff
@@ -63,7 +63,7 @@ export default class CreateAssetPackModal extends React.PureComponent<Props, Sta
       view = CreateAssetPackView.SUCCESS
     } else if (progress.stage !== ProgressStage.NONE && !error) {
       view = CreateAssetPackView.PROGRESS
-    } else if (error) {
+    } else if (view !== CreateAssetPackView.EXIT && error) {
```

Comment:
When clicking the close button the modal enters the exit state, showing the message to confirm you want to discard the asset package.
In the case of the error, it tried to go to the exit state but this function made it so that it went back to the edit asset pack state, making it seem like nothing happened